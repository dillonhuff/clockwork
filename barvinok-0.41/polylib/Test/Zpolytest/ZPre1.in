# Preimage by readMat["(i,j->3i+1,2i+5j)"] of readMat["(i,j->2i,2i+j)"]
M 2
3 3
2 0 0 
2 1 0
0 0 1
3 3
3 0 1
2 5 0
0 0 1
#readDom["{i,j| 1<= i,j<=10}"]
D 1
4 4
	  1 1 0 -1
          1 0 1 -1
          1 -1 0 100
          1 0 -1 100
F 11



