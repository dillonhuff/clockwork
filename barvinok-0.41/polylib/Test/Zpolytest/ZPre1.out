
ZPOLYHEDRON: Dimension 2 

LATTICE: 
3 3
    2    0    1
    0    1    0
    0    0    1
POLYHEDRON Dimension:2
           Constraints:4  Equations:0  Rays:4  Lines:0
Constraints 4 4
Inequality: [    3    0    1 ]
Inequality: [   -2    5   -3 ]
Inequality: [   -3    0   98 ]
Inequality: [    2   -5  102 ]
Rays 4 4
Vertex: [   98   41 ]/3
Vertex: [   -5    7 ]/15
Vertex: [   -5  304 ]/15
Vertex: [  490  502 ]/15
